src/api-service/__app__/requirements.txt

```diff
@@ -1,34 +1,34 @@
-azure-common~=1.1.25
```

Comment:
Please revert this change.  It's hard to tell if you actually changed anything here.

Reply:
Done!